packages/xrpl/src/models/transactions/common.ts

```diff
@@ -134,7 +134,9 @@ export function isIssuedCurrency(
  * @param input - The input to check the form and type of
  * @returns Whether the AuthorizeCredential is properly formed
  */
-function isAuthorizeCredential(input: unknown): input is AuthorizeCredential {
+export function isAuthorizeCredential(
```

Comment:
This function's name is too specific now that it is also being used for PDs

Reply:
Yes, it's `AcceptedCredentials`, not `AuthorizeCredentials` there. That's my point.

Reply:
@achowdhry-ripple  If I rename this to AcceptedCredentials, will it confuse the Credential's usage? Should I rename it to `isValidCredential` instead?

Reply:
@mvadari I'd like to discuss this issue offline. I'll merge this PR at the moment because explorer changes need the client library